packages/screens/Launchpad/LaunchpadApply/LaunchpadApplyScreen.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { StyleSheet, View } from "react-native";
 
-import { LaunchpadBanner } from "../components/LaunchpadBanner";
+import { LaunchpadBanner } from "./LaunchpadCreate/components/LaunchpadBanner";
```

Comment:
Why is it in a ``LaunchpadCreate`` folder ? Or should we rename it ``LaunchpadCreateBanner`` if it applies only to the create page ?

Reply:
![image](https://github.com/user-attachments/assets/d16f0b04-9b99-4c3e-a5a7-4f79ccbb06e3)

On the big Launchpad PR, I organized the files like that.
![image](https://github.com/user-attachments/assets/edb1a448-98da-4803-a359-fa3610c08e64)

LaunchpadBanner exists on origin/main, so I want to place it in the good folder to "win" a change on the big Launchpad PR. It's a preshot.

LaunchpadBanner is used in Launchpad Create screen that depends on Launchpad Apply screen